custom_components/tuya_local/devices/orison_chanfok_neo_fan_light.yaml

```diff
@@ -0,0 +1,102 @@
+name: Orison Chanfok Neo
+products:
+  - id: eba992532f93bb1c009bjw
+    name: Orison Chanfok Neo
+primary_entity:
+  entity: fan
+  dps:
+    - id: 107
+      type: boolean
+      name: switch
+    - id: 106
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: fresh
+          value: fresh
+        - dps_val: nature
+          value: nature
+    - id: 105
+      type: integer
+      name: speed
+      optional: true
+      range:
+        min: 1
+        max: 6
+    - id: 104
+      type: string
+      name: direction
+    - id: 103
+      type: integer
```

Comment:
This can be a sensor of class duration if you know the units (min or s).
Based on the name, it should probably be coupled with a method of setting the timer?

Reply:
> Does your fan have two LEDs? RGB facing the ceiling and white facing down???

Yes, and with this config I can control only the main light. This config is not fully implemented but at least the big things work (fan and main light)

Reply:
This reply does not seem to match with the comment. I think you are actually replying to another comment on the main PR by another user.